Data/ByteString.hs

```diff
@@ -889,6 +891,14 @@ take n ps@(BS x l)
     | otherwise = BS x n
 {-# INLINE take #-}
 
+-- | 'takeEnd' @n xs@ is equivalent to @'drop' ('length' xs - n) xs@.
```

Comment:
This is a fine statement, but I would appreciate adding a verbal explanation of the semantics. Please also mention computational complexity O(1), similar to other functions.